src/transforms/encryption.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const crypto = require('crypto')
+const { Transform } = require('stream')
+let aesjs
+
+if (crypto.getCiphers().indexOf('aes-128-cfb8') !== -1) {
+  // Native supported
+  module.exports.createCipher = function (secret) {
+    return crypto.createCipheriv('aes-128-cfb8', secret, secret)
+  }
+
+  module.exports.createDecipher = function (secret) {
+    return crypto.createDecipheriv('aes-128-cfb8', secret, secret)
+  }
+} else {
+  // aes-js fallback
+  aesjs = require('aes-js')
+
+  module.exports.createCipher = function (secret) {
+    return new Cipher(secret)
+  }
+
+  module.exports.createDecipher = function (secret) {
+    return new Decipher(secret)
+  }
+}
+
+class Cipher extends Transform {
+  constructor (key) {
+    super()
+    // eslint-disable-next-line
+    this.aes = new aesjs.ModeOfOperation.cfb(key, key, 8)
+  }
+
+  _transform (chunk, enc, cb) {
+    try {
+      cb(null, this.aes.encrypt(chunk))
```

Comment:
This won't work with a streaming input. Did you try it ?

Reply:
`aes-js` has a bug with CFB encryption (any other algorithms worked fine there)

Marking this PR as draft until they fix that

Reply:
Is there an open issue for this ?

Reply:
Yes, I'm also making PR